napalm_panos/panos.py

```diff
@@ -31,11 +31,12 @@
 
 from napalm_base.base import NetworkDriver
 from netmiko import ConnectHandler
+from netmiko import __version__ as netmiko_version
 
 
 class PANOSDriver(NetworkDriver):
 
-    def __init__(self, hostname, username, password, timeout=60, optional_args=None):
+    def __init__(self, hostname, username, password='', timeout=60, optional_args=None):
```

Comment:
Don't do this, please.Signature has to be the same in all the methods. I don't want someone to start writing code for `panos` think that the password field can be omited and then introduce another driver and realize the code is suddenly broken.